raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate_regression.py

```diff
@@ -366,7 +365,7 @@ def test_regression_mediator_task_no_routes():
         ),
         channelidentifiers_to_channels=channels.channel_map,
         pseudo_random_generator=pseudo_random_generator,
-        block_number=expired_block_number,
+        block_number=receiver_lock_expired,
```

Comment:
This value is different from the previous version, why did you change it from sender to receiver?

Reply:
The value was calculated for both the sender and receiver. 
The one on line 345 should be the value for sender while this one should be for the receiver (depending on the expectation of the test)

Reply:
Ok you're right... i figured i had it the other way around.